2020-07-24 14:02:28,771 ERROR: Exception on / [GET] [in c:\users\lenovo\desktop\microblog\python-projects\microblog\venv\lib\site-packages\flask\app.py:1892]
Traceback (most recent call last):
  File "c:\users\lenovo\desktop\microblog\python-projects\microblog\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\lenovo\desktop\microblog\python-projects\microblog\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\lenovo\desktop\microblog\python-projects\microblog\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\lenovo\desktop\microblog\python-projects\microblog\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\lenovo\desktop\microblog\python-projects\microblog\venv\lib\site-packages\flask\app.py", line 1948, in full_dispatch_request
    rv = self.preprocess_request()
  File "c:\users\lenovo\desktop\microblog\python-projects\microblog\venv\lib\site-packages\flask\app.py", line 2242, in preprocess_request
    rv = func()
  File "C:\Users\lenovo\Desktop\microblog\Python-Projects\Microblog\app\routes.py", line 18, in before_request
    g.locale = str(get_locale())
  File "c:\users\lenovo\desktop\microblog\python-projects\microblog\venv\lib\site-packages\flask_babel\__init__.py", line 251, in get_locale
    rv = babel.locale_selector_func()
  File "C:\Users\lenovo\Desktop\microblog\Python-Projects\Microblog\app\__init__.py", line 32, in get_locale
    return request.accept_languages.best_match(config['LANGUAGES'])
NameError: name 'config' is not defined
2020-07-24 14:02:36,700 ERROR: Exception on /favicon.ico [GET] [in c:\users\lenovo\desktop\microblog\python-projects\microblog\venv\lib\site-packages\flask\app.py:1892]
Traceback (most recent call last):
  File "c:\users\lenovo\desktop\microblog\python-projects\microblog\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\lenovo\desktop\microblog\python-projects\microblog\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\lenovo\desktop\microblog\python-projects\microblog\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\lenovo\desktop\microblog\python-projects\microblog\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\lenovo\desktop\microblog\python-projects\microblog\venv\lib\site-packages\flask\app.py", line 1948, in full_dispatch_request
    rv = self.preprocess_request()
  File "c:\users\lenovo\desktop\microblog\python-projects\microblog\venv\lib\site-packages\flask\app.py", line 2242, in preprocess_request
    rv = func()
  File "C:\Users\lenovo\Desktop\microblog\Python-Projects\Microblog\app\routes.py", line 18, in before_request
    g.locale = str(get_locale())
  File "c:\users\lenovo\desktop\microblog\python-projects\microblog\venv\lib\site-packages\flask_babel\__init__.py", line 251, in get_locale
    rv = babel.locale_selector_func()
  File "C:\Users\lenovo\Desktop\microblog\Python-Projects\Microblog\app\__init__.py", line 32, in get_locale
    return request.accept_languages.best_match(config['LANGUAGES'])
NameError: name 'config' is not defined
2020-07-24 14:03:04,870 INFO: Microblog startup [in C:\Users\lenovo\Desktop\microblog\Python-Projects\Microblog\app\__init__.py:60]
2020-07-24 14:03:07,252 ERROR: Exception on / [GET] [in c:\users\lenovo\desktop\microblog\python-projects\microblog\venv\lib\site-packages\flask\app.py:1892]
Traceback (most recent call last):
  File "c:\users\lenovo\desktop\microblog\python-projects\microblog\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\lenovo\desktop\microblog\python-projects\microblog\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\lenovo\desktop\microblog\python-projects\microblog\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\lenovo\desktop\microblog\python-projects\microblog\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\lenovo\desktop\microblog\python-projects\microblog\venv\lib\site-packages\flask\app.py", line 1948, in full_dispatch_request
    rv = self.preprocess_request()
  File "c:\users\lenovo\desktop\microblog\python-projects\microblog\venv\lib\site-packages\flask\app.py", line 2242, in preprocess_request
    rv = func()
  File "C:\Users\lenovo\Desktop\microblog\Python-Projects\Microblog\app\routes.py", line 18, in before_request
    g.locale = str(get_locale())
  File "c:\users\lenovo\desktop\microblog\python-projects\microblog\venv\lib\site-packages\flask_babel\__init__.py", line 251, in get_locale
    rv = babel.locale_selector_func()
  File "C:\Users\lenovo\Desktop\microblog\Python-Projects\Microblog\app\__init__.py", line 33, in get_locale
    return request.accept_languages.best_match(config['LANGUAGES'])
TypeError: 'module' object is not subscriptable
2020-07-24 14:04:38,830 INFO: Microblog startup [in C:\Users\lenovo\Desktop\microblog\Python-Projects\Microblog\app\__init__.py:59]
2020-07-24 14:05:27,192 INFO: Microblog startup [in C:\Users\lenovo\Desktop\microblog\Python-Projects\Microblog\app\__init__.py:59]
2020-07-24 14:15:47,533 INFO: Microblog startup [in C:\Users\lenovo\Desktop\microblog\Python-Projects\Microblog\app\__init__.py:59]
2020-07-24 14:16:25,348 INFO: Microblog startup [in C:\Users\lenovo\Desktop\microblog\Python-Projects\Microblog\app\__init__.py:59]
2020-07-29 17:16:14,286 INFO: Microblog startup [in C:\Users\lenovo\Desktop\microblog\Python-Projects\Microblog\app\__init__.py:59]
2020-07-29 17:38:17,937 INFO: Microblog startup [in C:\Users\lenovo\Desktop\microblog\Python-Projects\Microblog\app\__init__.py:59]
2020-07-29 17:38:31,911 INFO: Microblog startup [in C:\Users\lenovo\Desktop\microblog\Python-Projects\Microblog\app\__init__.py:59]
